### PR TITLE
Feature/cis k8s benchmark v1.5.0

### PR DESCRIPTION
**Pull request by aromeyer**
Trying to implement the CIS k8S Benchmark v1.5.0.

Many many evolutions in the document structure but few "deep" changes.
I try in the following table to summarize the biggest changes (sorry if I missed something):

| old ref | new ref | status |
| --- | --- | --- |
| cis-kubernetes-benchmark-1.1.3 | | DELETE |
| cis-kubernetes-benchmark-1.1.12 (flagged as deprecated) | | DELETE |
| | cis-kubernetes-benchmark-3.1.1 | NEW |
| | cis-kubernetes-benchmark-3.2 | NEW |
| cis-kubernetes-benchmark-2.1.11 (flagged as deprecated) | | DELETE |

### CODE REVIEW

File: controls/3_1_control_plane_authn_and_authz.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+#
```
Comment (chris-rock) [inline controls/3_1_control_plane_authn_and_authz.rb]: Is the copyright header intended here?

Comment (aromeyer) [reply controls/3_1_control_plane_authn_and_authz.rb]: Nothing intended on my side, I was just working on files from "stupid" copy paste.
This header can be changed to suit what you find most appropriate.

Concerning the ids, I was just following what is now the section organisation in the Kubernetes CIS Benchmark 1.5 document (Control plane is a new section compared to 1.4).

I think the benchmark implementation should follow the document structure it refers to.
As you said, avoiding breaking changes can be managed by publishing new releases each time it is needed. So users using/looking for the X.Y version/implementation will always be able to get the benchmark implementation from a release while you can continue to follow the  benchmark evolution and publish new releases at any time.

Comment (chris-rock) [reply controls/3_1_control_plane_authn_and_authz.rb]: Agreed. Let's release a new version. Can you please remove the apache header? We use a common license file in the root directory and individual commits need to be signed off.

Comment (aromeyer) [reply controls/3_1_control_plane_authn_and_authz.rb]: The header has been removed from all the ruby files.

**Comment by aromeyer** [comment]
Thanks for the merge.
Don't worry it was not a high priority topic for me and I totally understand that it is the same for you.

**Pull request merged**